src/galaxy-integrations/galaxy-integrations.ts

```diff
@@ -16,6 +16,10 @@ export const DefaultGalaxyIntegrations = {
     pathPrefix: "/events",
     appName: "TAMU Datathon 2020 Events"
   }),
+  "/mailing": newGalaxyIntegrationConfig({
+    pathPrefix: "/mailing",
+    appName: "TAMU Datathon 2020 Mailing"
```

Comment:
```suggestion
    appName: "TD Mailing List"
```

Nitpick: Since this isn't specifically for TAMU Datathon 2020 (it's for us to use internally), the name should really be more generic.

Reply:
Also pls ignore the map file